pubspec.yaml

```diff
@@ -54,8 +54,8 @@ dependencies:
   package_info_plus: ^8.0.0
   path: ^1.8.3
   path_provider: ^2.0.13
-  share_plus: ^9.0.0
-  share_plus_platform_interface: ^4.0.0
+  share_plus: ^10.0.0
+  share_plus_platform_interface: ^5.0.0
```

Comment:
Done by editing pubspec.yaml and running `tools/upgrade pub` twice.

Huh, why'd you have to run `tools/upgrade` twice? I'd expect it to be idempotent — for the first time to go all the way that a second time would take you. (Modulo new package versions getting uploaded to the package repo in between.)

Reply:
(Linking the follow-up to this on CZO [here](https://chat.zulip.org/#narrow/channel/243-mobile-team/topic/deps.3A.20running.20.60tools.2Fupgrade.20pub.60.20twice))